pandas/util/testing.py

```diff
@@ -2042,40 +2039,31 @@ def package_check(pkg_name, version=None, app='pandas', checker=LooseVersion,
     try:
         have_version = mod.__version__
     except AttributeError:
-        raise exc_failed_check('Cannot find version for %s' % pkg_name)
+        pytest.skip('Cannot find version for %s' % pkg_name)
     if checker(have_version) < checker(version):
-        raise exc_failed_check(msg)
+        pytest.skip(msg)
 
 
 def skip_if_no_package(*args, **kwargs):
-    """Raise SkipTest if package_check fails
+    """pytest.skip() if package_check fails
 
     Parameters
     ----------
     *args Positional parameters passed to `package_check`
     *kwargs Keyword parameters passed to `package_check`
     """
-    from nose import SkipTest
-    package_check(exc_failed_import=SkipTest,
-                  exc_failed_check=SkipTest,
-                  *args, **kwargs)
+    package_check(*args, **kwargs)
 
 
 def skip_if_no_package_deco(pkg_name, version=None, app='pandas'):
```

Comment:
yeah I don't know how to actually test if a tests is actually skipped; in nose you can catch the ``SkipTest``, but for pytest you would never do this.